packages/netsuite-adapter/src/change_validators/unique_fields.ts

```diff
@@ -164,14 +206,18 @@ const validateDuplication = (
   return awu(changesData)
     .map(change => ({
       elemID: change.elemID,
-      field: getters.getChangeRestrictedField(change),
+      fields: Array.from(
+        new Set(
+          getters.getChangeRestrictedFields(change).filter(field => uniqueFieldToID[field] > 1)
+        )
+      ),
```

Comment:
Another way to do it- decide which one you prefer
```suggestion
      fields: _.uniq(getters.getChangeRestrictedFields(change).filter(field => uniqueFieldToID[field] > 1)),
```